test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -603,6 +603,11 @@ public ColumnFamilyStore getCurrentColumnFamilyStore(String keyspace)
              : Keyspace.open(keyspace).getColumnFamilyStore(currentTable);
```

Comment:
Should be replaced by `getColumnFamilyStore`

Reply:
I'm sorry, I don't get - what should be replaced?

Reply:
I think that this should be replaced by a call to the new `CQLTester#getColumnFamilyStore` method right below:
```suggestion
             : getColumnFamilyStore(keyspace, currentTable);
```